index.html

```diff
@@ -1703,65 +1748,51 @@ <h2>Verification Methods</h2>
+value MUST be an <a data-cite="INFRA#ordered-set">ordered set</a> of
+verification methods, where each verification method is described by a
+<a data-cite="INFRA#ordered-map">map</a>. The <a>verification method</a>
+<a data-cite="INFRA#ordered-map">map</a> MUST include the <code>id</code>,
+<code>type</code>, <code>controller</code>, and specific verification method
+properties as determined by the value of <code>type</code>, and MAY include
+additional properties.
           </p>
-
           <p>
-In the case where a verification method is a public key, the value of the
-<code><a>id</a></code> property might be structured as a
-<a href="https://en.wikipedia.org/wiki/Compound_key">compound key</a>. This is
-especially useful for integrating with existing key management systems and key
-formats such as JWK [[RFC7517]]. It is RECOMMENDED that JWK <code>kid</code>
-values are set to the public key fingerprint [[RFC7638]]. It is RECOMMENDED
-that verification methods that use JWKs to represent their public keys utilize
-the value of <code>kid</code> as their fragment identifier. See the first key
-in <a href="#example-16-various-verification-method-types"></a>
-for an example of a public key with a compound key identifier.
+The value of the <code><a>id</a></code> property for a verification method
+MUST be a <a data-cite="INFRA#string">string</a> that conforms to the rules in
+Section <a href="#did-url-syntax"></a>. When more than one verification method
+is present, the value of <code><a>verificationMethod</a></code> MUST NOT
+contain multiple entries with the same <code>id</code>. If the value of
+<code><a>verificationMethod</a></code> contains multiple entries with the same
+<code>id</code>, a <a>DID document</a> processor MUST produce an error.
           </p>
 
-          <p>
-The value of the <code>type</code> property MUST be exactly one verification
-method type. In order to maximize global interoperability, the
-<a>verification method</a> type SHOULD be registered in the
-[[?DID-SPEC-REGISTRIES]].
-          </p>
           <p>
 The value of the <code>controller</code> property MUST be a <a
 data-cite="INFRA#string">string</a> that conforms to the rules in Section <a
 href="#did-syntax"></a>.
           </p>
+          <p>
+The value of the <code>type</code> property MUST be exactly one verification
+method type expressed as a <a data-cite="INFRA#string">string</a>. In order to
+maximize interoperability, the <a>verification method</a> type SHOULD be
+registered in the DID Specification Registries [[?DID-SPEC-REGISTRIES]].
+          </p>
 
           <p>
-A verification method MUST contain verification material, such as
-<a>publicKeyJwk</a> or <a>publicKeyBase58</a>.
+A verification method MUST contain a property to express verification
+material. The property used is determined by the value of the
+<code>type</code> property. Examples of such properties are
+<code><a>publicKeyJwk</a></code> or <code><a>publicKeyBase58</a></code>.
```

Comment:
The terminology "such properties are ..." made clear a different error in the table before (and in the SHACL and SHEX constraints as well as the JSON Schema). Indeed, in all those cases the constraints say that _either_ a `publicKeyJwk` _or_ a `publicKeyBase58`, but one of the two must be present.

To avoid such a misunderstanding, it may be worth adding a reference to [§4.4. of the registry document](https://www.w3.org/TR/did-spec-registries/#verification-method-properties) to refer to other possible properties that can be used instead.